stuff/i18n.py

```diff
@@ -98,8 +116,11 @@ def _get_translated_strings(self, contents_callback):
                         (row.strip(), filename, lineno + 1),
                         fixable=False)
 
-        self._check_missing_entries(strings, languages)
-        return strings
+        # Removing badges entries
+        strings_without_badges = {
```

Comment:
los nombres de variable son excelentes para ayudarle al lector a saber qué está ocurriendo, y en ese sentido casi siempre es preferible tener un buen nombre de variable a tener un comentario.

en este caso en particular, únicamente estás usando la variable para regresar el valor, y como tener ambos {el nombre bonito, el comentario} es redundante, lo mejor es eliminar uno:

```suggestion
        return {
```

en general el patrón

```python
    x = ...
    return x
```

casi siempre es mejor escribirlo como

```python
    return ...
```